### DISCUSSION TITLE
Multiple Rejections Despite All Required Evidence Submitted

### DISCUSSION THREAD

**Discussion by pa-0** [community/community#141363] [GitHub Education]
### Hi everyone,

So far, I have verified / submitted proof of:

- [x] valid `edu` email verification;
- [x] school transcript (dated) -- clearly legible with all required elements visible/unobstructed;
- [x] student ID with University name, student ID, student photo all clearly visible;
- [x] my billing information is accurate; and,
- [x] my full name, school name, student status, student ID, and dates are all clearly legible and fit the size/dimension requirements. 

I'm not sure if it's even worth the effort at this point after so many failed attempts at the appication.

**Comment by t1noo7** [comment]
Hi @pa-0,

How about the status of your application?

- the green label: Your request was approved and your benefits will be available soon -> so you have to wait for the verification. It will cost you a few days, or even a couple of weeks; depending on different factors, such as the volume of requests.
- the purple one: Your request was approved and your benefits are now available -> you can now access it. Go to the [pack](https://education.github.com/pack) -> go down to the "All Offers" section and select the available one as your desired

Otherwise, you can create a ticket [here](https://support.github.com/contact/education) if you need further help, choose a `having trouble redeeming my Global Campus offers` option to go forward your issue.

Drop by a [FAQs](https://github.com/orgs/community/discussions/111352) here, hope this helps you better!

**Reply by pa-0** [reply]
Hi @t1noo7  Thanks for the info!  So I followed the link to create a ticket, and when selecting the applicable issue topic, I was met with this popup (and the Subject line of the issue form was disabled):

![image](https://github.com/user-attachments/assets/b85673a9-2354-4457-a68d-482825fdaf95)

  My application status says Rejected in a red badge. Checking my history, it's more like 8-10 attempts as opposed to what I estimated earlier.

**Reply by t1noo7** [reply]
Seems like you got an automatic notification from the system, about the option `Application Not Approved (student)` is no longer supported at the moment with your account, try another by opening a ticket [here](https://support.github.com/contact/account), choose `Other`, tell them everything you did, everything you have the correct with, and you can also ask more about the account status in the ticket.. Some GitHub Staff will answer you there, although it may take a several time, need to be patient.

**Reply by pa-0** [reply]
Thanks, will do.
